static/js/drafts.js

```diff
@@ -65,6 +66,11 @@ export const draft_model = (function () {
         drafts[id] = draft;
         save(drafts);
 
+        // Show draft introduction when the first draft is created.
+        if ($("#hotspot_intro_draft_overlay").length) {
+            hotspots.open_popover("intro_draft");
+        }
```

Comment:
This is messy.  I think we want this to just have a single line that calls a `hotspots.show_drafts_popover_if_needed()` function, and then the internals can be in the hotspots.js logic.

Reply:
I think so creating a special function for every non-intro hotspot is not a good idea. 

I have edited the `hotspots.open_popover` -> `hotspots.open_popover_if_hotspot_exist` to handle this situation (see https://github.com/zulip/zulip/pull/17557/commits/14514d9832df8c68b1a53c425c3b0c598266da9e). 

Which will make the above code one-liner.

If you don't like the idea, I am happy to add `hotspots.show_drafts_popover_if_needed`.

P.S: I am not sure about the name of the function maybe it should be `hotspots.open_popover_if_exist`?